src/core/PeakController.cpp

```diff
@@ -80,9 +80,7 @@ void PeakController::updateValueBuffer()
 {
 	if( m_coeffNeedsUpdate )
 	{
-		const float ratio = 44100.0f / Engine::audioEngine()->outputSampleRate();
-		m_attackCoeff = 1.0f - powf( 2.0f, -0.3f * ( 1.0f - m_peakEffect->attackModel()->value() ) * ratio );
-		m_decayCoeff = 1.0f -  powf( 2.0f, -0.3f * ( 1.0f - m_peakEffect->decayModel()->value()  ) * ratio );
+		m_coeff = 100.0f / Engine::audioEngine()->outputSampleRate();
```

Comment:
The old ratio variable gave me the idea. I noticed that it had hard coded 44100.0 and thought it was odd. Since they needed the sample rate to calculate the ratio, why use 44100.0? Wasn't 44100.0 just the sample rate * 0.01?

The rest of the old formula is incomprehensible to me. But since they were previously using the sample rate in their coefficient, I wondered if the buzzing was caused by the coefficient being wrong.

I tried a few things and ultimately found that this worked, but it necessitated removing attack/decay (hence why I moved them). At least moving them ended up helping with the stair step problem as well 😀 

Thanks for the review!

Reply:
Oh, also for linear interpolation, you want your coefficient to be between 0 and 1, so dividing 100 by the sample rate seemed a decent way to get a number of that sixe while also using the sample rate in the formula.